checksource-gen/app/Main.hs

```diff
@@ -17,15 +20,19 @@ main :: IO ()
 main = do
   pj@PlanJson {..} <- findAndDecodePlanJson
         $ ProjectRelativeToDir "/home/runner/submission"
-  let rootUnit = (pjUnits Map.!) $ head $ Set.toList $ planJsonIdRoots pj
+  let rootUnit = (pjUnits Map.!)
+        . fromJust
+        . L.find (\(Plan.UnitId text) -> T.isPrefixOf (T.pack "submission") text)
+        . Set.toList
+        $ planJsonIdRoots pj
```

Comment:
I'm running the makefile to how this change helps, but it's taking time, so please explain if you have some time available!

Reply:
Excuse I'm late! When we add `ac-library-hs`, it comes with an executable and causes an error here.

Reply:
Not at all! Thanks for being quick.